python/tank_vendor/shotgun_authentication/ui/aspect_preserving_label.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from .qt_abstraction import QtGui
+from .qt_abstraction import QtCore
+
+
+class AspectPreservingLabel(QtGui.QLabel):
```

Comment:
Am I right thinking that this class is not needed for the current window, which always uses a fixed sg logo? I think we should remove the code. If, at some point in the future, we need to host different logos, we can always add it back.